Hymns/Favorites/FavoritesViewModel.swift

```diff
@@ -0,0 +1,23 @@
+import SwiftUI
+import RealmSwift
+import Combine
+
+//Mixture of Combine and Realm notifications to push db updates to the UI
+//Hard to find resources for this so here is a link to the foreign site to understand implementation.
+//https://qiita.com/chocoyama/items/af172b32f492b706c96d#observableobject%E3%82%92%E5%88%A9%E7%94%A8%E3%81%99%E3%82%8B
+
+class FavoritesViewModel: ObservableObject {
+    var objectWillChange = ObservableObjectPublisher()
+    private(set) var itemEntities: Results<FavoritedEntity> = RealmHelper.all()
```

Comment:
Hmmm. I don't think we need the favorites list to update in real-time, so it's probably sufficient to just load a list of Favorites and just display that. But if that's difficult to do, we can keep this

Reply:
I tried to take it out but the reason I think it is still worth keeping is if you go into a favorited hymn from the favorited list and remove it then immediately come back to the favorited screen it takes a second to refresh and you see it disappear. With the publisher update it happens right away.

Reply:
ok, sounds good